store/snapshots/manager.go

```diff
@@ -484,4 +488,56 @@ func (m *Manager) snapshot(height int64) {
 
 		m.logger.Debug("pruned state snapshots", "pruned", pruned)
 	}
+
+	if os.Getenv("SNAPSHOT_EXPORT_ENABLED") == "1" {
```

Comment:
Having a random environment variable seems like a bad UX IMHO. We do not do that anywhere in the SDK.

Reply:
CLI flag is a definitely better choice, I'll update it